### PR TITLE
Material theme tooltip adjust

### PR DESCRIPTION
**Pull request by Olexi**

### CODE REVIEW

**Review by jsdmitry — DISMISSED** [review]

File: js/viz/core/themes/material.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ registerTheme({
         paddingTopBottom: 6,
         arrowLength: 0,
         location: "edge",
-        color: "#616161",
+        color: "#fff",
```
Comment (babich-a) [inline js/viz/core/themes/material.js]: White text on the white background - it is the worst idea I have ever seen. Right color described in the documentation https://material.io/design/components/tooltips.html#spec.

File: styles/widgets/material/color-schemes/material.light.less [hunk 2]
```diff
@@ -1458,7 +1458,7 @@
 * @type color
 * @group overlays.tooltip
 */
-@tooltip-bg: #616161;
+@tooltip-bg: rgba(255, 255, 255, 1);
```
Comment (babich-a) [inline styles/widgets/material/color-schemes/material.light.less]: Look at my comment about charts' tooltip. Material tooltip background should be #616161.

**Review by babich-a — CHANGES_REQUESTED** [review]

File: styles/widgets/material/scheduler.material.less [hunk 3]
```diff
@@ -387,8 +383,10 @@
 }
 
 .dx-scheduler-dropdown-appointments {
+    margin-top: @MATERIAL_SCHEDULER_DROPDOWN_DAY_MARGIN;
     position: absolute;
     &.dx-button {
+        border-radius: 12px;
```
Comment (Yarovaya) [inline styles/widgets/material/scheduler.material.less]: Describe some const for this property

File: styles/widgets/material/tooltip.material.less [hunk 4]
```diff
@@ -10,12 +10,32 @@
...
+                margin-bottom: 0;
+                margin-top: 0;
+                .dx-tooltip-appointment-item {
+                    .dx-tooltip-appointment-item-marker-body {
+                        @MATERIAL_TOOLTIP_MARKER_DIMENSION: 24px;
+                        width: @MATERIAL_TOOLTIP_MARKER_DIMENSION;
+                        height: @MATERIAL_TOOLTIP_MARKER_DIMENSION;
+                    }
+                    .dx-tooltip-appointment-item-content-subject {
+                        margin-top: 4px;
```
Comment (Yarovaya) [inline styles/widgets/material/tooltip.material.less]: Describe some consts for properties

**Review by Yarovaya — CHANGES_REQUESTED** [review]

File: styles/widgets/material/tooltip.material.less [hunk 5]
```diff
@@ -13,8 +13,8 @@
         box-shadow: none;
         border-radius: @tooltip-border-radius;
         min-height: auto;
-        .dx-popup-content {
```
Comment (babich-a) [inline styles/widgets/material/tooltip.material.less]: Redundant change.

File: styles/widgets/material/popup.material.less [hunk 6]
```diff
@@ -27,7 +27,7 @@
 @MATERIAL_POPUP_TITLE_BOTTOM_MARGIN: @MATERIAL_POPUP_CONTENT_WITHTITLE_TOP_PADDING - @MATERIAL_POPUP_CONTENT_PADDING;
 
 .dx-popup-wrapper {
-    & > .dx-overlay-content {
+    &.dx-overlay-wrapper > .dx-overlay-content {
```
Comment (babich-a) [inline styles/widgets/material/popup.material.less]: This change brokes dxTooltip. If we need to change the tooltip background color in scheduler we need to use scheduler-specific classes.

File: styles/widgets/material/scheduler.material.less [hunk 7]
```diff
@@ -901,3 +915,45 @@
         }
     }
 }
+.dx-tooltip-wrapper .dx-overlay-content .dx-popup-content {
+    padding: 0;
```
Comment (babich-a) [inline styles/widgets/material/scheduler.material.less]: You remove the paddings for dxTooltip. Does scheduler's tooltip have some specific class?

**Review by babich-a — CHANGES_REQUESTED** [review]

File: styles/widgets/generic/scheduler.generic.less [hunk 8]
```diff
@@ -659,3 +659,7 @@
 .dx-scheduler-agenda-nodata {
     font-size: @GENERIC_SCHEDULER_WORKSPACE_INFO_TEXT_SIZE;
 }
+
+.dx-scheduler-appointment-tooltip-wrapper .dx-overlay-content .dx-popup-content .dx-list-item {
```
Comment (babich-a) [inline styles/widgets/generic/scheduler.generic.less:663]: By the way it is generic

**Review by babich-a — APPROVED** [review]

File: styles/widgets/generic/scheduler.generic.less (hunk 8 above)
Comment (Olexi) [reply styles/widgets/generic/scheduler.generic.less:663]: There was a feedback from Dmitriy Kharkov for generic theme tooltip in scheduler

**Review by Yarovaya — APPROVED** [review]

**Pull request merged**